crates/iceberg/src/arrow/schema.rs

```diff
@@ -35,6 +35,9 @@ use rust_decimal::prelude::ToPrimitive;
 use std::collections::HashMap;
 use std::sync::Arc;
 
+/// When iceberg map type convert to Arrow map type, the default map field name is "key_value".
+pub(crate) const DEFAULT_MAP_FIELD_NAME: &str = "key_value";
```

Comment:
May I ask what's the reason of changing `entries` to `key_value`? :eyes:

Reply:
From https://github.com/apache/iceberg-python/blob/9b9ed534b2022cb9a687f4ed876fadcc2457b31b/pyiceberg/io/pyarrow.py#L2197, we can see that the field name of map type in the column name path of parquet is `key_value`